spec/manipulate_file_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'manipulate_file'
+
+describe '#get_first_line' do
+  it 'read the first line of the file' do   
+    @content = "Test 1,2,3"
+
+    arq = File.new("test.log","w")
+    arq.puts @content
+    arq.close
+
+    main = ManipulateFile.new(arq)
+
+    expect(main.get_first_line).to eq(@content)
+  end
+    
+  it 'treat error when file doesn`t exist' do
+    arq = "teste"
+    main = ManipulateFile.new(arq)
+
+    expect(main.file).to eq(nil)
```

Comment:
Faça com que o teu código gere uma exceção de fato, e dai teste a não existencia do arquivo com: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher